examples/sdk-examples/src/examples/attributeFilter/customAttributeFilter/UseAttributeFilterHandler.tsx

```diff
@@ -58,7 +58,7 @@ export function CustomAttributeFilter(props: ICustomAttributeFilter) {
                     <div
                         key={element.uri}
                         onClick={() => {
-                            if (isElementSelected(element.uri)) {
+                            if (isElementSelected(element.uri!)) {
```

Comment:
The ! should not be necessary, the `isElementSelected` supports null (maybe it needs to support undefined too?)

Reply:
fixed